Gems/Atom/RHI/Code/Include/Atom/RHI.Reflect/Limits.h

```diff
@@ -8,6 +8,7 @@
 #pragma once
 
 #include <Atom/RHI.Reflect/Base.h>
+#include <AzCore/Casting/numeric_cast.h>
```

Comment:
does this need casting?
```suggestion
#include <AzCore/std/limits.h>
```